website/docs/guides/migration/versions/02-upgrading-to-v1.5.md

```diff
@@ -77,3 +77,6 @@ More to come!
 
 ### dbt-core Python API
 - Auto-generated documentation ([#2674](https://github.com/dbt-labs/docs.getdbt.com/issues/2674)) for dbt-core CLI & Python API for programmatic invocations
+
+### Quick hits
```

Comment:
This is a silly name for the section header — but I figure I'll rename it, and add in more of the small stuff, during v1.5 guide revision (#3170)